src/components/SignUpForm/index.js

```diff
@@ -11,17 +10,36 @@ import useValidation from 'hooks/useValidation';
 import useTextInputProps from 'hooks/useTextInputProps';
 import signUpValidations from 'validations/signUpValidations';
 import ErrorView from 'components/common/ErrorView';
-import styles from './styles';
+import Button from 'components/common/Button';
+
+import { BLACK } from 'constants/colors';
+
+import Picker from 'components/common/Picker';
 
 const FIELDS = {
+  name: 'name',
   email: 'email',
   password: 'password',
   passwordConfirmation: 'passwordConfirmation',
+  gender: 'gender',
+};
+
+const PLACEHOLDER_PICKER = {
+  label: strings.SIGN_UP.picker.label.toUpperCase(),
+  value: '',
+  color: BLACK,
 };
 
+const GENDER_LABELS = [
+  { label: strings.SIGN_UP.picker.male, value: 'male' },
+  { label: strings.SIGN_UP.picker.female, value: 'female' },
+  { label: strings.SIGN_UP.picker.other, value: 'other' },
+];
+
```

Comment:
I think you can build the whole picker functionality with the `useSelectOptions()` custom hook inside the project

Reply:
you can take a look at those files as a guidance 

https://github.com/Stukz/target-onboarding/blob/master/src/components/SignUpForm/index.js

https://github.com/Stukz/target-onboarding/blob/master/src/components/common/Picker/index.js

Reply:
It sound interesting but I can't see the repo. Is it private?